# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/contextualidentity/test/browser/browser_saveLink.js
# Commit: 39f2e21623aa
# Full Hash: 39f2e21623aa7c19517e32cbd97dc1c3584c58ca
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-02 03:24:46
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 2c873c72bf1f -r 39f2e21623aa browser/components/contextualidentity/test/browser/browser_saveLink.js
--- a/browser/components/contextualidentity/test/browser/browser_saveLink.js	Mon Jun 01 16:49:05 2020 +0000
+++ b/browser/components/contextualidentity/test/browser/browser_saveLink.js	Mon Jun 01 16:49:03 2020 +0000
@@ -13,7 +13,13 @@
 
   // make sure userContext is enabled.
   await SpecialPowers.pushPrefEnv({
-    set: [["privacy.userContext.enabled", true]],
+    set: [
+      ["privacy.userContext.enabled", true],
+      // This test does a redirect from https to http and it checks the
+      // cookies. This is incompatible with the cookie sameSite schemeful
+      // feature and we need to disable it.
+      ["network.cookie.sameSite.schemeful", false],
+    ],
   });
 });
 